examples/app/BUILD.bazel

```diff
@@ -46,20 +47,25 @@ http_server(
     templated_args = ["package"],
 )
 
-ts_config(
-    name = "tsconfig-test",
-    src = "tsconfig-test.json",
-    deps = [":tsconfig.json"],
-)
-
-ts_library(
+# we could use ts_library here, but we use plain typescript to demonstrate that it works
+tsc(
     name = "e2e",
     testonly = 1,
-    srcs = ["app.e2e-spec.ts"],
-    tsconfig = ":tsconfig-test",
-    deps = [
+    outs = [
+        "app.e2e-spec.js",
+    ],
+    args = [
+        "-p",
+        "$(location tsconfig-test.json)",
+        "--outDir",
```

Comment:
comment explaining what `--outDir $@` expands to and why it is needed may be good for the purposes of this example

Reply:
yes, also added a sentence to the `npm_package_bin` docs